components/demo/demo-snippet.js

```diff
@@ -9,6 +9,7 @@ class DemoSnippet extends LitElement {
 		return {
 			codeViewHidden: { type: Boolean, reflect: true, attribute: 'code-view-hidden' },
 			fullscreen: { type: Boolean, reflect: true },
+			fullscreenView: { type: Boolean, reflect: true, attribute: 'fullscreen-view' },
```

Comment:
Looks like this property should be private, ya? If so, then I think this should be defined like:
```javascript
_fullscreenView: { state: true }
```
And conditionally render an internal class to apply the styles instead of using `:host([fullscreen-view])`. Generally, we prefer not sprouting private stuff on the host.